packages/fxa-settings/src/components/Settings/MfaGuard/error-boundary.tsx

```diff
@@ -62,9 +63,6 @@ export class MfaErrorBoundary extends Component<
         this.props.sessionToken,
         this.props.requiredScope
       );
```

Comment:
If the error is not thrown here, it won't bubble up. What ensures we dont have a 'silent' failure now that the throw has been removed?